app/scrapers/__init__.py

```diff
@@ -42,6 +42,8 @@ def feed_gen(query, engine, count=10, qtype=''):
     engine = old_names.get(engine, engine)
     if engine in ('quora', 'youtube'):
         urls = scrapers[engine].search_without_count(query)
+    elif engine in ('mojeek',) and qtype == 'news':
```

Comment:
Why use `in ('mojeek',)`
You can use `if engine == 'mojeek':`
Since there is only one option to choose from

Reply:
@gabru-md Made the requested changes :+1: 
Please review again.